surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/logger/SurefireProviderLoggerFactory.java

```diff
@@ -23,7 +23,7 @@ public Logger getLogger() {
 
     private class SurefireProviderLogger implements Logger {
 
-        private static final String PREFIX = "%s: Smart Testing Extension - ";
+        private static final String PREFIX = "%s: Smart Testing Provider - ";
```

Comment:
Please see this comment: https://github.com/arquillian/smart-testing/pull/152#discussion_r140217015